R/g.part5.addsib.R

```diff
@@ -61,6 +61,7 @@ g.part5.addsib = function(ts, epochSize, part3_output, desiredtz, sibDefinition,
     redo1 = nightsi[1] - ((60/epochSize)*60) # 1 hour before first midnight
     if (redo1 < 1) redo1 = 1
     redo2 = nightsi[1] + (14*(60/epochSize)*60) # 14 hours after first midngiht
+    if (redo2 > Nts) redo2 = Nts
```

Comment:
I just merged all in this PR. The exporting of the short time series file still works. 

Note that this line only checks that redo2 is not beyond the end of the measurement because the line before potentially incorrectly assumes that there is always 14 hours extract after the first night. If it is then the new line clips redo2 to the number of rows in ts (Nts). So, this should not do any harm.

@jhmigueles Would you mind testing this branch again with the files you used to test #892 to make sure that functionality is not broken?

Reply:
Just did it and everything works well.